sdk/devcenter/azure-developer-devcenter/azure/developer/devcenter/_operations/_operations.py

```diff
@@ -2891,7 +3022,8 @@ def _delete_dev_box_initial(
     @distributed_trace
     def begin_delete_dev_box(
         self, project_name: str, user_id: str, dev_box_name: str, **kwargs: Any
-    ) -> LROPoller[JSON]:
+    ) -> LROPoller[_models.OperationDetails]:
```

Comment:
Just checking is this the expected return type?

Reply:
Yes, this is correct